Dockerfile

```diff
@@ -1,15 +1,28 @@
-FROM ipython/scipystack
+FROM ubuntu:14.04
 
-ADD . /srv/runner/
-WORKDIR /srv/runner/
-
-RUN pip2 install .
+MAINTAINER Kyle Kelley <rgbkrk@gmail.com>
 
+RUN apt-get update && apt-get install -y wget bzip2
 RUN useradd -m -s /bin/bash rho
 
+RUN echo 'export PATH=/home/rho/anaconda:$PATH' > /etc/profile.d/conda.sh
```

Comment:
I like to combine consecutive `RUN` lines into a single instruction joined with `\` so that we create fewer layers.

It's not actually a big deal, though.